codebuild/bin/s2n_codebuild.sh

```diff
@@ -63,6 +63,10 @@ if [[ "$OS_NAME" == "osx" && "$TESTS" == "integration" ]]; then
     scan-build --status-bugs -o /tmp/scan-build make -j$JOBS; STATUS=$?; test $STATUS -ne 0 && cat /tmp/scan-build/*/* ; [ "$STATUS" -eq "0" ];
 fi
 
+# Run Multiple tests on one flag.
+if [[ "$TESTS" == "ALL" || "$TESTS" == "sawHMACPlus" ]] && [[ "$OS_NAME" == "linux" ]]; then make -C tests/saw tmp/verify_HMAC.log tmp/verify_drbg.log sike failure-tests; fi
+
+# Run Individual tests
```

Comment:
Will the SAW tests get run multiple times if the "ALL" flag is set?

Reply:
good catch- I removed the "ALL" from the individual tests being combined.